### ISSUE TITLE
Add these things

### ISSUE THREAD

**Issue by SoniEx2** [MightyPirates/OpenComputers#2647]
![dual processor](https://upload.wikimedia.org/wikipedia/commons/e/e4/Motherboard-dual-processor.png)

(https://commons.wikimedia.org/wiki/File:Motherboard-dual-processor.png by Toniperis, CC-BY-SA 3.0)

Basically can we get a Minecraft version of this real-life thing called a "dual-processor motherboard"?

**Comment by payonel** [comment]
@SoniEx2 you're abusing our github system. I've told you we are not increasing our cpu nor memory options for open computers. Please discontinue this

**Comment by SoniEx2** [comment]
dual processors are pretty common for servers IRL. can we please get that in MC?

**Comment by payonel** [comment]
no, please stop

**Comment by Vectrobe** [comment]
isn't the whole idea of having server racks so that you can set up multiple servers connected to the one bus? it doesn't exactly work any different to use separate systems synchronised via an event bus than using separate threads. Of which by the way, lua doesn't actively support multithreading, so these computers technically cant support multiple CPU cores anyway without significant changes to the lua specification...

TLDR; just use server racks how they're supposed to be used...

**Comment by SoniEx2** [comment]
They'd increase single-threaded performance, automatically increasing multi-green-threaded performance.

**Comment by Vectrobe** [comment]
no... that's not how that works...

**Comment by SoniEx2** [comment]
It's a *game*
